src/lib/Microsoft.Health.Events/EventConsumers/Service/EventBatchingService.cs

```diff
@@ -73,6 +77,34 @@ public void NewPartitionInitialized(string partitionId)
             {
                 _eventPartitions.TryRemove(partitionId, out _);
             }
+
+            // if _scaledOutMaxEventsAllPartitions is set then assume horizontal scaling is enabled
```

Comment:
Suggestion: We might want to have a more direct signal or at least modify the comment.  This absolutely makes since for the FHIR Conversion but I believe we are still using the EventBatchingService for normalization as well but have a very small batch size and I don't see that changing as we scale out since we don't get much benefit with batching for events.

Reply:
updated the comment. yeah we will probably not set this environment variable for normalization